pkg/bindings/containers/types_execstartandattach_options.go

```diff
@@ -26,53 +22,35 @@ func (o *ExecStartAndAttachOptions) Changed(fieldName string) bool {
 // ToParams
 func (o *ExecStartAndAttachOptions) ToParams() (url.Values, error) {
 	params := url.Values{}
+
 	if o == nil {
 		return params, nil
 	}
-	json := jsoniter.ConfigCompatibleWithStandardLibrary
-	s := reflect.ValueOf(o)
-	if reflect.Ptr == s.Kind() {
-		s = s.Elem()
+
+	if o.OutputStream != nil {
```

Comment:
Those options structs are supposed to be send over wire, right? How can one serialize `io.Writer/Reader`?

Reply:
They are not supposed to be send over wire. see: https://github.com/containers/podman/blob/afe4ce6b1ce6a5ba21472f51defd243bdd7750b3/pkg/bindings/containers/attach.go#L500-L519

Reply:
@Luap99  So only some struct fields (structurally simple ones) are serialized. The other may be handled differently (e.g. post as JSON in body). How it's decided? Is it solely based on type. For instance `int` fields are include in uri params, but struct/interface are not?

Reply:
I have no idea. @baude wrote the bindings stuff